site/content@beta/agent-chat-rtm-api/scopes.md

```diff
@@ -0,0 +1,116 @@
+---
+weight: 20
+---
+
+# Scopes
+
+Scopes define the access to certain resources.
+
+- `ro` means read only
+- `rw` means read/write
+- all `rw` scopes contain `ro` privileges
+- all `access` scopes contain `my` privileges
+- all `all` scopes contain `access` privileges and `my` privileges
+
+
+For more details check [here](../services/sso/typedef.go).
+
+## Accessing chat 
+
+This chart shows every possible chats scenarios:
+
+- chats without my access and without my presence (`chats -A -P`)
+- chats with my access but without my presence (`chats +A -P`)
+- chats with my presence but without my access (`chats -A +P`)
+- chats with my access and with my presence (`chats +A +P`)
+
+![](./chats.png)
+
+The table shows scopes dependency of accessing chat:
```

Comment:
Nie jestem pewna co autor miał na myśli przez to zdanie. Ta tabela to jest powtórzenie rysunku. Tu nie chodziło czasem o scopes capabilities of accessing chats (zamiast dependency)?